.github/workflows/test.yaml

```diff
@@ -1,4 +1,4 @@
-name: 'Test OpenAPI Spec Diff Action'
+name: "Test OpenAPI Spec Diff Action"
```

Comment:
why are you prefer using " over '?
see github example: https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action

Reply:
That's Prettier doing its thing. I thought I had undone most of its "damage". I'll fix it up.